packages/ddd-toolkit/src/command-bus/local-command-bus.ts

```diff
@@ -16,12 +16,15 @@ export class LocalCommandBus implements ICommandBus {
         this.retryMechanism = new ExponentialBackoff(retryInitialDelay);
```

Comment:
retry is not used yet?

Reply:
we can remove it or add a TODO and a card in project

Reply:
@lucagiove Retry is used in the non-synchronous send. I don't understand, do you want to remove the non-synchronous send?

Reply:
@gtoselli my bad the diff hide the rest of the code I missed the usage of the retry. It's perfect sorry.